google-cloud-bigtable/lib/google/cloud/bigtable/table_data_operations.rb

```diff
@@ -0,0 +1,483 @@
+# frozen_string_literal: true
+
+require "google/gax"
+require "google/cloud/bigtable/v2"
+require "google/cloud/bigtable/chunk_reader"
+require "google/cloud/bigtable/flat_row"
+require "google/cloud/bigtable/mutation_entry"
+require "google/cloud/bigtable/rows_reader"
+
+module Google
+  module Cloud
+    module Bigtable
+      GRPC_RETRYABLE_ERRORS = [
+        GRPC::DeadlineExceeded,
+        GRPC::Aborted,
+        GRPC::Unavailable,
+        GRPC::Core::CallError
+      ].freeze
+
+      DEFAULT_READ_RETRY_COUNT = 3
+
+      class TableDataOperations
+        attr_reader :client, :table_path, :app_profile_id
+
+        def initialize client, table_path, app_profile_id = nil
```

Comment:
This method should be marked private. It looks like only way to get this object is via `DataClient#table`, and the `client` argument here is a low-level GAPIC object that should not be part of the public API.

Reply:
ok, I will make change for it.